test/test.js

```diff
@@ -128,4 +128,47 @@ describe('beaver-logger tests', () => {
             }
         });
     });
+
+    it('should not send event to server if enableServerLogging is false', () => {
+        const $logger = Logger({
+            url:                  '/test/api/log',
+            disableServerLogging: true
+        });
+
+        const browserConsoleLogs = [];
+
+        window.console.info = (args) => browserConsoleLogs.push(args);
+
+        $logger.info('browser_only_log', {
+            foo: 'bar',
+            bar: true
+        });
+
+        let handlerCalled = false;
+
+        const logEndpoint = $mockEndpoint.register({
+            method:  'POST',
+            uri:     '/test/api/log',
+            handler: () => {
+                handlerCalled = true;
+                return {};
+            }
+        });
+
+        logEndpoint.expectCalls();
+        return $logger.flush().then(() => {
+            try {
+                logEndpoint.done(); // will throw is not API calls received by logEndpoint
```

Comment:
`// will throw if API calls received by logEndpoint`

Reply:
https://github.com/krakenjs/sync-browser-mocks/blob/master/README.md#xmlhttprequest 

it('should correctly call /api/user', function() {

    myListener.expectCalls();
    
    // Run some code which would call the api

    myListener.done(); // This will throw if the api was not called
});

I can set a variable in the catch block to indicate that an exception is indeed thrown.

Reply:
👍

Reply:
Done